src/engine/readaheadmanager.cpp

```diff
@@ -160,7 +160,7 @@ SINT ReadAheadManager::getNextSamples(double dRate, CSAMPLE* pOutput,
             }
         }
 
-        if (crossFadeSamples) {
+        if (crossFadeSamples >= 0) {
```

Comment:
still trying to figure when `crossFadeSamples < 0` occurs.
if I get it right this can happen only if the overshoot (fractional frames?) corrects the curr. play pos AND we're looping over the track start, so that `loop_read_position > crossFadeStart`?
so we actually thow away the overshoot, no?